doc/usecases/gamification/ingredient_lists.md

```diff
@@ -0,0 +1,27 @@
+# Use Case: Ingredient Lists
+
+## Summary
+
+- **Scope:** Ingredient Lists
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can create lists to keep track of ingredients needed over the year.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app and is editing a map.
+- **Main success scenario:**
+  - The user has a list of fruits and vegetables that are needed continuously over the course of a timespan, e.g. a daily smoothie or salad.
+  - The user inputs the items into a form in the map editor and can set a name and picture for the ingredient list.
+  - The user is able to have multiple of those ingredient lists.
+  - The lists will display what ingredients are available on the map and which still need to be planted by the user.
+  - The user is awarded every month with some [permacoins](./collaborating_member.md) for having all neccessary ingredients for a list.
```

Comment:
I don't think we should make milestones repeatable on a monthly basis. We can make a Blossom that is achievable for having all ingredients for a list in any month and/or multiple months, making it repeat for every season. But resetting every month takes the purpose of an achievement to be proud of away imo. That's why I would award permacoins directly if we want to make it a continuous source.

Reply:
Permacoins should be only awarded for contributions relevant for others, not what you do for yourself. We also shouldn't mix Permacoins into several use cases. Better to see it as a completely separate use case, as it is completely irrelevant for non-contributing members.

Reply:
Either way, my point stands that I wouldn't make the Blossoms repeatable every month. I would then simply state, that there are Blossoms that have 'having all necessary ingredients for a list' as an objective.

Reply:
What about awarding more or bigger blossoms for longer streaks (look for streak running if you don't know the term)?

Reply:
Please fix the remains here so that we can finally merge this.

Reply:
> What about awarding more or bigger blossoms for longer streaks (look for streak running if you don't know the term)?

I guess GitHub swallowed my comment that I originally gave last week as an answer, I am sorry for not noticing earlier.
If my memory does not betray me, we already spoke about making Blossoms with multiple 'checkpoints'. It would definitely make sense to specify the 'having all necessary ingredients for a list'-Blossom as such one, so we could award additional Blossom Upgrades (and therefore more permacoins for achieving those Blossoms) for longer streaks.

Reply:
That sounds great!